src/Misc/layoutbin/powershell/Add-AzureGuestAgentCapabilities.ps1

```diff
@@ -0,0 +1,24 @@
+[CmdletBinding()]
+param()
+
+$guestAgentPath = $null
+
+try {
+    $proc = Get-Process 'WindowsAzureGuestAgent'
```

Comment:
@chshrikh how does it matter? if he is an admin - he should be able to get this capability. if he is not an admin, he won't be able to.